### PR TITLE
refactor/1133 - Improve InputController class

### LINKED ISSUE #1133: Improve InputController class
Improve the `InputController` class.

### PR DESCRIPTION
**Pull request by clscott42**
Resolves #1133.

Improve `InputController` class.

### CODE REVIEW

File: src/assets/scripts/client/InputController.js [hunk 1]
```diff
@@ -765,6 +767,23 @@ export default class InputController {
         ]);
     }
 
+    /**
+     *
```
Comment (n8rzz) [inline src/assets/scripts/client/InputController.js]: Missing description

Comment (felixscheffer) [reply src/assets/scripts/client/InputController.js]: I added a description but if you can come up with something better, go for it.

**Comment by erikquinn** [comment]
Oops, I already said that. :joy:

**Comment by felixscheffer** [comment]
Should be working now.

**Comment by erikquinn** [comment]
Just tried on https://openscope-dev-pr-1134.herokuapp.com/ and no dice. 😕

**Comment by felixscheffer** [comment]
That's strange. What exactly isn't working?

**Comment by erikquinn** [comment]
Basically all input controller functions. Scrolling to zoom, clicking to click, typing letters to snap cursor into the input box, pressing enter to execute a command while in the input box, right clicking to drag... oddly though I've seen no console errors whilst trying stuff.

**Comment by erikquinn** [comment]
Disregard! Caching issue. The latest build does work correctly.

**Pull request merged**